frontend/src/app/shared/services/auth/auth.service.ts

```diff
@@ -56,7 +56,6 @@ export class AuthService {
             // We need to subscribe here because the request does not get triggered if we don't
             this.http.get('/user/login', 'outcobra')
                 .catch(err => {
-                    this.notificationService.remove(); //remove all notifications
```

Comment:
So the user can see the server-error message too (No connection, internal error, etc.)
I can remove them again if you think thats better